### PR TITLE
[DTM] - Documentation changes for CD 1020.1.10

### PR DESCRIPTION
**Pull request by Vagdevi1998**
* User documentation for CD 1020.1.10

### CODE REVIEW

File: content/dtm/asset-types-bundle/asset-properties.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ In the **Asset properties** simple asset properties are listed with their type.
...
+A complex asset property consists of multiple key-value pairs, meaning, it contains a root object followed by one or more key-value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key-value pair as well as providing a type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. You can expand each key-value pair to see a specific property. When you add a new key-value pair, it automatically expands.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/asset-properties.md:275]: Add a line to explain how user can collapse or expand a row to hide or view the property details

File: static/images/dtm/assets/dtm-assets-move-assets.png
Comment (lra-sag) [inline static/images/dtm/assets/dtm-assets-move-assets.png:1]: Why is this screenshot updated?

File: static/images/dtm/custom-property/dtm-property-library-import.png
Comment (lra-sag) [inline static/images/dtm/custom-property/dtm-property-library-import.png:1]: Why is this screenshot updated? This and the one above are not relevant to changes in complex properties

**Review by lra-sag — CHANGES_REQUESTED** [review]

File: static/images/dtm/assets/dtm-assets-move-assets.png
Comment (Vagdevi1998) [reply static/images/dtm/assets/dtm-assets-move-assets.png:1]: This screenshot is updated to reflect the button name changes for Import assets -> Import in the background.

File: static/images/dtm/custom-property/dtm-property-library-import.png
Comment (Vagdevi1998) [reply static/images/dtm/custom-property/dtm-property-library-import.png:1]: This screenshot is updated to reflect the button name change of import in the background.

File: content/dtm/asset-types-bundle/asset-properties.md (hunk 1 above)
Comment (Vagdevi1998) [reply content/dtm/asset-types-bundle/asset-properties.md]: Added.

File: content/dtm/asset-types-bundle/asset-properties.md [hunk 2]
```diff
@@ -272,7 +272,7 @@ In the **Asset properties** simple asset properties are listed with their type.
...
+A complex asset property consists of multiple key-value pairs, meaning, it contains a root object followed by one or more key-value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key-value pair as well as providing a type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. On the right side of each key-value pair, an expand icon <i class="dlt-c8y-icon-expand-arrow text-muted icon-20"></i> is available. On click of expand icon <i class="dlt-c8y-icon-expand-arrow text-muted icon-20"></i> you can see a specific property. On click of collapse icon <i class="dlt-c8y-icon-collapse-arrow text-muted icon-20"></i> you can hide a specific property. When you add a new key-value pair, it automatically expands.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-properties.md:275]:
```suggestion
A complex asset property consists of multiple key-value pairs, meaning, it contains a root object followed by one or more key-value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key-value pair as well as providing a type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. On the right side of each key-value pair, you see an expand icon <i class="dlt-c8y-icon-expand-arrow text-muted icon-20"></i>. Click the expand icon <i class="dlt-c8y-icon-expand-arrow text-muted icon-20"></i> to see a specific property. Note that when you add a new key-value pair, it automatically expands. To hide it, click the collapse icon <i class="dlt-c8y-icon-collapse-arrow text-muted icon-20"></i>.
```

**Comment by MWindel** [comment]
@Vagdevi1998 This screenshot (static/images/dtm/custom-property/dtm-property-library-import.png) shows way to much empty (grey) space. Please crop the screenshot to only the dialog window.

File: static/images/dtm/custom-property/dtm-property-library-import.png
Comment (MWindel) [reply static/images/dtm/custom-property/dtm-property-library-import.png:1]: @Vagdevi1998 Please consider to reduce the number of screenshots if they are not absolutely necessary for the reader to understand the content of your documentation.

**Comment by Vagdevi1998** [comment]
> @Vagdevi1998 This screenshot (static/images/dtm/custom-property/dtm-property-library-import.png) shows way to much empty (grey) space. Please crop the screenshot to only the dialog window.

Updated the screenshot

File: static/images/dtm/assets/dtm-assets-move-assets.png
Comment (lra-sag) [inline static/images/dtm/assets/dtm-assets-move-assets.png:1]: width of the image should be 1440px as per the standard please modify

File: static/images/dtm/custom-property/dtm-property-library-import.png
Comment (lra-sag) [inline static/images/dtm/custom-property/dtm-property-library-import.png:1]: crop the image closer to the popup such that it is at the center

**Review by lra-sag — CHANGES_REQUESTED** [review]
Go through all the screenshots for name change and add it in one go. Make sure to create a separate PR for each functionality to be documented. I believe the screenshots that show the Import button should have been updated as part of its corresponding documentation.

File: static/images/dtm/assets/dtm-assets-move-assets.png
Comment (MWindel) [reply static/images/dtm/assets/dtm-assets-move-assets.png:1]: @Vagdevi1998 @lra-sag The image does not need to be 1440px wide, but your SCREEN dimensions (defined under settings) need to be 1440x900.

In this case, the screenshot needs to be modified, depending on what you want to showcase. If the dialog window is the important bit, then please crop the screenshot to the dialog window.

File: static/images/dtm/custom-property/dtm-property-library-import.png
Comment (Vagdevi1998) [reply static/images/dtm/custom-property/dtm-property-library-import.png:1]: Modified

**Review by lra-sag — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**